app/components/form-contextual-key-value/template.hbs

```diff
@@ -0,0 +1,76 @@
+{{#if keyValuePairs.length}}
```

Comment:
What does this component look like in an edit state? Seems like we'd show the inputs still? Perhaps using `input-or-edit` to wrap the inputs would work for you here? Disregard if there is no edit state for this component.

Reply:
The component only supports editing right now. I didn't bother with a view state because I didn't think it would get used before the transition.

If you'd like me to add it because you know the view state is necessary or you'd like it for consistency sake let me know and I'll add it.

Reply:
Okay, that makes sense. I'm not going to push it to be in before I merge this then.